desktop/core/src/desktop/js/apps/storageBrowser/StorageBrowserPage/StorageBrowserTable/StorageBrowserTable.tsx

```diff
@@ -91,29 +91,28 @@ const StorageBrowserTable = ({
   ...restProps
 }: StorageBrowserTableProps): JSX.Element => {
   const [loadingFiles, setLoadingFiles] = useState<boolean>(false);
-  const [tableHeight, setTableHeight] = useState<number>();
+  const [tableHeight, setTableHeight] = useState<number>(100);
```

Comment:
`tableHeight` is the state of type number and initialising it will `undefined` would not a good option. I believe `100` would be good option to show small table when page is loading.
Your thoughts?

Reply:
Looking at the code it seems that this tableHeight is actually defining the table's initial "scroll area".

I think the whole table size logic needs an overhaul (where to place default widths and heights, proper naming based on actual function, how to trigger proper resizing using ResizeObserver etc) but lets do that in another PR.